reportserver/dao/DatabaseHandler.py

```diff
@@ -10,55 +10,60 @@
 from common.GlobalConfig import Configuration
 from reportserver.manager import dateTimeUtility
 
-cfg_path = os.getenv('RECCE7_PLUGIN_CONFIG') or 'config/plugins.cfg'
```

Comment:
just changing this code to be a class so that the configuration instance isn't created prematurely to allow for testing.